rmgweb/database/views.py

```diff
@@ -590,6 +1039,284 @@ def parseSoluteDataComment(comment):
+            else:
+                # rounding is not needed for experimental values from RMG-database
+                if solute_estimator != 'expt':
+                    if abs(value) < 1000:
+                        value = "{:.4f}".format(value)  # round to 4 decimal places
+                    else:
+                        value = "{:.2e}".format(value)  # display in scientific notation if the value is big
+            solute_data_results[key].append(value)
+            if solute_estimator == 'SoluteML' and key != 'V':
+                key_epi_unc = f'{key} epi. unc.'
+                if key_epi_unc in solute_epi_unc_dict:
+                    epi_unc_value = solute_epi_unc_dict[key_epi_unc]
+                    if abs(epi_unc_value) < 1000:
+                        epi_unc_value = "{:.4f}".format(epi_unc_value)  # round to 4 decimal places
+                    else:
+                        epi_unc_value = "{:.2e}".format(epi_unc_value)  # display in scientific notation if the value is big
+                    solute_data_results[key_epi_unc].append(epi_unc_value)
+                else:
+                    solute_data_results[key_epi_unc].append('-')
+
+        solute_data_list.append(solute_data)
+        solute_data_found_list.append(found)
+
+    # get the solvation properties if the input solvent was given
+    solvent_info = None
+    if solvent != 'None':
+        solute_data_results[f'dGsolv298({energy_unit})'] = []  # solvation free energy
+        solute_data_results[f'dHsolv298({energy_unit})'] = []  # solvation enthalpy
+        solute_data_results[f'dSsolv298({energy_unit}/K)'] = []  # solvation entropy
+
+        additional_info_list.append('dGsolv298: solvation free energy at 298 K.')
+        additional_info_list.append('dHsolv298: solvation enthalpy at 298 K.')
+        additional_info_list.append('dSsolv298: solvation entropy at 298 K.')
+
+        # get the solvent data and href
+        solvent_entry = database.solvation.libraries['solvent'].entries[solvent]
+        solvent_data = solvent_entry.data
+        solvent_smiles_list = []
+        for spc in solvent_entry.item:
+            solvent_smiles_list.append(spc.smiles)
+        solvent_index = solvent_entry.index
+        solvent_href = reverse('database:solvation-entry',
+                               kwargs={'section': 'libraries', 'subsection': 'solvent',
+                                       'index': solvent_index})
+        solvent_info = (solvent, solvent_smiles_list, solvent_href, solvent_index)
+
+        # check whether we have all solvent parameters to calculate dGsolv and dHsolv
+        abraham_parameter_list = [solvent_data.s_g, solvent_data.b_g, solvent_data.e_g, solvent_data.l_g,
+                                  solvent_data.a_g, solvent_data.c_g]
+        dGsolv_availble = not any(param is None for param in abraham_parameter_list)
```

Comment:
typo? should be "available" for this and the `dHsolv_available` below.

Reply:
Thanks for catching this. It's fixed in 3cb7caf.